go.mod

```diff
@@ -10,7 +10,7 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20201214114959-164a2fb63b5f // OpenShift 4.7
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49 // OpenShift 4.7
```

Comment:
Do we need to replace this with 4.8?

Reply:
I am unable to update the api library:
```
[sebsoto@sdesktop windows-machine-config-operator]$ go get -u github.com/openshift/api@be21c6cb1bea11427ff13ef95e155aa04b7c5939
github.com/openshift/api imports
	k8s.io/api/batch/v2alpha1: cannot find module providing package k8s.io/api/batch/v2alpha1
```

Reply:
Did you trying deleting the two lines and running `go mod tidy && go mod vendor && go mod verify`?

```patch
diff --git a/go.mod b/go.mod
index 2767e881..12156208 100644
--- a/go.mod
+++ b/go.mod
@@ -7,8 +7,6 @@ replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503
 replace (
        github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
        github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
-       github.com/openshift/api => github.com/openshift/api v0.0.0-20201214114959-164a2fb63b5f // OpenShift 4.7
-       github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49 // OpenShift 4.7
 
        k8s.io/api => k8s.io/api v0.22.0-rc.0
        k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.0-rc.0
```

Reply:
Doing that results in picking up commits from may, when we want the latest release-4.8 commits
```
[sebsoto@sdesktop windows-machine-config-operator]$ go mod tidy && go mod vendor && go mod verify
go: downloading github.com/openshift/api v0.0.0-20210521075222-e273a339932a
go: downloading github.com/openshift/client-go v0.0.0-20210521082421-73d9475a9142
all modules verified
```

Reply:
I am fine with that. Better than release-4.7. We can revisit bumping to the latest in a separate PR.